### DISCUSSION TITLE
PR #2050 missed the templating url_for function

### DISCUSSION THREAD

**Discussion by rosstitmarsh** [encode/starlette#2126] [Potential Issue] [answered]
PR #2050 fixed the url_for functions not allowing "name" as a parameter, but it missed the templating url_for function. https://github.com/encode/starlette/blob/01aa49a379520d3bbe6bc1aecc48a48169e6b004/starlette/templating.py#L81

I have a PR ready.

**Comment by Kludex** [comment, marked as answer]
Push it, please.

**Reply by rosstitmarsh** [reply]
PR #2127
